### DISCUSSION TITLE
NLogger support

### DISCUSSION THREAD

**Discussion by ashishneosoftmail** [NeoSOFT-Technologies/rest-dot-net-core#48] [Ideas]
NLogger support

**Comment by sasachan** [comment]
We shall go for [Serilogger](https://github.com/NeoSOFT-Technologies/rest-dot-net-core/discussions/27) based implementation over NLogger or anything

https://github.com/NeoSOFT-Technologies/rest-dot-net-core/discussions/27

![image](https://user-images.githubusercontent.com/12516416/128685588-1643dc28-b7e2-48a9-b90e-53dae1ece679.png)

**Reply by sasachan** [reply]
@ashishneosoftmail - thanks for your input
